src/commands/graphql-repos.js

```diff
@@ -82,6 +81,7 @@ function getRepositoryObject(data) {
 }
 
 exports.handler = async ({ file }, state) => {
+	const { bizOps, bizOpsErrorHandler } = require('../lib/bizops')
```

Comment:
**suggestion**: `require` is event-loop blocking, and if we wanted to migrate to ECMAScript Modules, this wouldn't work, so using `await import(...)` might be better:
```suggestion
	const { bizOps, bizOpsErrorHandler } = await import('../lib/bizops.js')
```

Reply:
On second thoughts - this is going to prove tricky - first we'd need to upgrade eslint and install babel-parser (fine) but second jest does not want to work with mixed cjs and esm, I paired with Serena for a while and eventually agreed it wasn't worth the investment involved. Any thoughts? @apaleslimghost

Reply:
ah, fair enough!